program-runtime/src/loaded_programs.rs

```diff
@@ -583,11 +583,6 @@ impl LoadingTaskWaiter {
 struct SecondLevel {
```

Comment:
This structure could be dissolved now again. It was only introduced to hold the `cooperative_loading_lock` after all.

Reply:
Good point I'll nuke it

Reply:
done